cooked-validators/src/Cooked/Currencies.hs

```diff
@@ -0,0 +1,47 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+
+module Cooked.Currencies where
+
+import qualified Ledger
+import qualified Ledger.Contexts as Validation
+import qualified Ledger.Typed.Scripts as Scripts
+import qualified Ledger.Value as Value
+import qualified PlutusTx
+import PlutusTx.Prelude hiding (Applicative (..))
+
+-- * QuickValue Minting Policy
+
```

Comment:
Quick values are extensively documented in the `Wallet` module. Perhaps it's worth linking the docs here, or moving the docs here altogether, along with the helpers, since the user might be rather inclined to find info about specific values in a module named `Currencies`.

Reply:
I think all the quick/permanent values helpers still in `Wallet` should be centralized here too.